spec/integration/plan_spec.rb

```diff
@@ -74,6 +74,24 @@
          '--no-host-key-check']
       }
 
+      context 'output' do
+        let(:output)    { StringIO.new }
+        let(:outputter) { Bolt::Outputter::Human.new(true, true, true, true, output) }
+        before(:each) do
+          allow(Bolt::Outputter::Human).to receive(:new).and_return(outputter)
+        end
+        it 'outputs message with verbose flag' do
+          run_cli(%W[plan run output::verbose --targets #{target} --verbose] + config_flags,
+                  outputter: Bolt::Outputter::Human)
+          expect(output.string).to match(/Hi, I'm Dave/)
+        end
+        it 'doesnt output without verbose flag' do
+          result = run_cli(%W[plan run output::verbose --targets #{target}] + config_flags,
+                           outputter: Bolt::Outputter::Human)
+          expect(result).not_to match(/Hi, I'm Dave/)
+        end
```

Comment:
```suggestion
        let(:output)    { StringIO.new }
        let(:outputter) { Bolt::Outputter::Human.new(true, true, true, true, output) }

        before(:each) do
          allow(Bolt::Outputter::Human).to receive(:new).and_return(outputter)
        end

        it 'outputs message with verbose flag' do
          run_cli(%W[plan run output::verbose --targets #{target} --verbose] + config_flags,
                  outputter: Bolt::Outputter::Human)
          expect(output.string).to match(/Hi, I'm Dave/)
        end

        it 'doesnt output without verbose flag' do
          result = run_cli(%W[plan run output::verbose --targets #{target}] + config_flags,
                           outputter: Bolt::Outputter::Human)
          expect(result).not_to match(/Hi, I'm Dave/)
        end
```

Spacing these out improves readability.